minitext.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name = "minitext"
-  s.version = "0.0.6"
+  s.version = "0.0.7.avvo"
```

Comment:
Can we change this to just 0.0.7? Or don't change the version in the PR at all and I'll version before I release. Either way works for me.